File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.js
function solution(phone_number) {
    var answer = "";
    for(let i = 0; i < phone_number.length-4; i++){
        answer += '*';
    }for(let i = phone_number.length-4; i < phone_number.length; i++){
        answer += phone_number[i];
    }
    return answer;
}//이렇게 풀면 안되는것같은데

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.cs
public class Solution {
    public string solution(string phone_number) {
        char[] cr = new char[phone_number.Length];
        for(int i = 0; i < phone_number.Length-4;i++){
            cr[i] = '*';
        }
        for(int i = phone_number.Length-4; i < phone_number.Length; i++){
            cr[i] = phone_number[i];
        }
        string answer = new string(cr);
        return answer;
    }
} 